### ISSUE TITLE
New pre-trained model

### ISSUE THREAD

**Issue by henrycjh** [IDEA-Research/OSX#19]
Hi, thanks for updating. Is the new pre-trained model named osx_l_wo_decoder.pth.tar fine-tuned on UBody? Why is it smaller than the original one?

**Comment by ZhengdiYu** [comment]
> Hi, it's the pretrained model for the setting of `--encoder_setting osx_l --decoder_setting --wo_decoder`. We add it because it's more efficient and we'll add more pretrained models in the future for a better trade-off between efficiency and accuracy.

Hi, is this model much worse than the original `osx_l.pth.tar`? Or is it not compatible with `demo.py`? I tried to change the settign in demo.py to use this osx_l_wo_decoder.pth.tar. I only change decoder setting from `normal` to `--decoder_setting --wo_decoder`, but the results are completely wrong with the wo_decoder model. Just like below:

![image](https://user-images.githubusercontent.com/63605407/233849863-9e753c42-1f7d-4325-ae2e-4f18e1d77a63.png)
![image](https://user-images.githubusercontent.com/63605407/233849847-855bced5-cf88-4ce7-b125-b9d1d1bdbc3e.png)
The front one is using wo_decoder model
